File: shipPet/src/main/java/com/ship/model/dto/DoctorDto.java
package com.ship.model.dto;

import com.ship.model.entity.Doctor;

public class DoctorDto extends Doctor {
    private String departmentName;

    public String getDepartmentName() {
        return departmentName;
    }

    public void setDepartmentName(String departmentName) {
        this.departmentName = departmentName;
    }
}

File: ship-ui/src/model/DO/DoctorDto.ts
/**
 * 单个医生
 */
import {Doctor} from "@/model/entiy/Doctor";

export interface DoctorDto extends Doctor {
    departmentName: string;
}

/**
 * 医生个人中心
 */
export const exampleDoctorOne = (): DoctorDto => {
    return {
        "doctorId": 0,
        "departmentId": null,
        "username": "",
        "doctorName": "",
        "doctorGender": true,
        "doctorAge": "",
        "doctorPhoto": "",
        "doctorTel": "",
        "authLv": 0,
        "doctorInfo": "",
        "doctorPassword": null,
        "createTime": null,
        "updateTime": null,
        "departmentName": ""
    }
}